tests/mock/clitools.rs

```diff
@@ -279,6 +279,24 @@ pub fn expect_ok_eq(config: &Config, args1: &[&str], args2: &[&str]) {
     }
 }
 
+pub fn expect_component_executable(config: &Config, cmd: &str) {
+    let out1 = run(config, cmd, &["--version"], &[]);
+    if !out1.ok {
+        print_command(&[cmd, "--version"], &out1);
+        println!("expected.ok: true");
+        panic!()
+    }
+}
+
+pub fn expect_component_not_executable(config: &Config, cmd: &str) {
+    let out1 = run(config, cmd, &["--version"], &[]);
+    if out1.ok {
+        print_command(&[cmd, "--version"], &out1);
+        println!("expected.ok: true");
```

Comment:
Should this be `false`?

Reply:
Yes, that's a copy/paste hiccough.  Fixed.